src/Polly/Bulkhead/BulkheadSyntax.cs

```diff
@@ -52,7 +52,7 @@ public static BulkheadPolicy Bulkhead(int maxParallelization, int maxQueuingActi
     /// <param name="onBulkheadRejected">An action to call, if the bulkhead rejects execution due to oversubscription.</param>
     /// <returns>The policy instance.</returns>
     /// <exception cref="ArgumentOutOfRangeException">maxParallelization;Value must be greater than zero.</exception>
-    /// <exception cref="ArgumentOutOfRangeException">maxParallelization;Value must be greater than zero.</exception>
+    /// <exception cref="ArgumentOutOfRangeException">maxParallelization;Value must be greater than or equal to zero.</exception>
```

Comment:
This is wrong, and without the edit is a duplicate of the line above. Did you mean for it to apply to `maxQueuingActions`?

Reply:
Yep, changed reference to `maxQueuingActions`.